pkg/multus/multus.go

```diff
@@ -258,16 +258,25 @@ func confDel(rt *libcni.RuntimeConf, rawNetconf []byte, multusNetconf *types.Net
 	return err
 }
 
-func conflistAdd(rt *libcni.RuntimeConf, rawnetconflist []byte, multusNetconf *types.NetConf, exec invoke.Exec) (cnitypes.Result, error) {
+func conflistAdd(rt *libcni.RuntimeConf, rawnetconflist []byte, cniConfList *libcni.NetworkConfigList, multusNetconf *types.NetConf, exec invoke.Exec) (cnitypes.Result, error) {
```

Comment:
TODO: Check that this works without conflists

Reply:
So, I think it works as intended without conflists... If you don't have a conflist to begin with -- you can't chain on it.

But in that  case, you can use `AuxiliaryCNIChainName` and have that as a place to chain instead.